ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_dialog/.content.xml

```diff
@@ -279,6 +279,17 @@
                                                     </cui>
                                                 </uiSettings>
                                             </thankYouMessage>
+                                            <dorAbsenceMessage
+                                            granite:class="warning-icon"
+                                            granite:id="dorWarningIcon"
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="/libs/fd/af/components/dorAbsenceBanner"
+                                            text="Use a non-Rapid Development Environment to use or test a Submit actions that requires the Document of Record feature"
+                                            variant="minimal">
+                                            <granite:rendercondition
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="/libs/fd/af/dor/rendercondition/dorAbsence"/>
```

Comment:
ditto

Reply:
resource type should not start with /libs or /apps, it should be relative to support overlay

Reply:
Removed /libs